### PR TITLE
Foundant glm slm source addition

### PR DESCRIPTION
**Pull request by aszalacinski**
- removes duplicate source file called glmslm
- updates original source file of glm

### CODE REVIEW

File: sources/glm.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-  "id": "GLM",
-  "name": "Grant Lifecycle Manager",
+  "id": "GLMSLM",
```
Comment (pfrisella) [inline sources/glm.json:2]: Keep the ID the same since it's already in use.

**Review by pfrisella — CHANGES_REQUESTED** [review]

File: sources/glm.json (hunk 1 above)
Comment (aszalacinski) [reply sources/glm.json]: fixed

**Comment by aszalacinski** [comment]
The change request was completed 11 days ago. Can this be merged now? Please advise.

**Comment by pfrisella** [comment]
Closing due to inactivity.

**Pull request closed without merging**